python_modules/dagster/dagster/_core/definitions/partitions/subset/partitions_subset.py

```diff
@@ -18,7 +18,7 @@ class PartitionsSubset(ABC, Generic[T_str]):
 
     @property
     def is_empty(self) -> bool:
-        return len(list(self.get_partition_keys())) == 0
+        return len(self) == 0
```

Comment:
nit: should this be moved to be by the `__len__` abstract method ?

Reply:
no opinion, doesn't len call __len__ under the hood?